### DISCUSSION TITLE
v5 -> v6 upgrade path for `--global`?

### DISCUSSION THREAD

**Discussion by rulatir** [pnpm/pnpm#3351] [Q&A] [answered]
I upgraded to v6, and certain `pnpm --global update <package>` commands that previously succeeded now error out with:

```
 ERROR  No package.json found
```

What is the correct upgrade path?

**Comment by zkochan** [comment, marked as answer]
Run `pnpm -g root` to see the location of your global package.

pnpm v5 was installing to `pnpm-global/4`. You can rename that folder to `pnpm-global/5` and run `pnpm install -g`
